react/features/filmstrip/components/web/Toolbar.js

```diff
@@ -68,26 +60,8 @@ class Toolbar extends Component<Props> {
      * otherwise.
      */
     _shouldShowButton(buttonName) {
-        return this.props._visibleButtons.has(buttonName);
+        return visibleButtons.has(buttonName);
     }
 }
 
-/**
- * Maps (parts of) the redux state to the associated props for this component.
- *
- * @param {Object} state - The Redux state.
- * @private
- * @returns {{
- *     _visibleButtons: Set<string>
- * }}
- */
-function _mapStateToProps(state): Object { // eslint-disable-line no-unused-vars
-    // XXX: We are not currently using state here, but in the future, when
-    // interfaceConfig is part of redux we will.
-
-    return {
-        _visibleButtons: new Set(interfaceConfig.TOOLBAR_BUTTONS)
-    };
-}
-
-export default connect(_mapStateToProps)(Toolbar);
+export default Toolbar;
```

Comment:
I don't like this change. It goes backwards to our general direction of not using config globals.

Reply:
Why not do the same thing you did in Toolbox.js?

Reply:
Because it doesn't need connect. I mean here we have 2 choices:
 - Don't use connect. Which I've done!
 - Put interface_config in redux finally as the comment states. Which is kind of different task!

I don't see what we gain if we put the redundant connect there. If we put interface_config in redux one day, it is obvious that we need actually read it from there in this component too.

Reply:
It does need to connect, eventually. What you are suggesting is we undo what needs to be done and do it again later, which is pointless. Please leave it as it was and include your optimization of using the sama array referene.